web/js/phoneNumberView.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
+!(function (global) {
+  var input;
+
+  var sendPhoneNumber = function () {
+    var phoneNumber = input.intlTelInput('getNumber');
+    Utils.sendEvent('phoneNumberView:dialOut', {
```

Comment:
You can pass as detail directly the phoneNumber avoiding to create a wrapper object when it is not required

```js
Utils.sendEvent('phoneNumberView:dialOut', phoneNumber);
var phoneNumber = evt.detail; // In the code where the event is dispatched
```